addon/core/handler.ts

```diff
@@ -64,6 +65,20 @@ export default class TableHandler {
       });
   }
 
+  async fetchColumnDefinitions(): Promise<ColumnDefinition[]> {
+    this.loadingColumnDefinition = true;
+
+    return this.tableManager
+      .fetchColumnDefinitions()
+      .then(({ column_definitions }) => {
+        this.columnDefinitions = column_definitions;
+        return column_definitions;
```

Comment:
do you even need to return ? 🤷‍♂️ you can return a Promise<void> i think

Reply:
Yes you are right, I change that to `Promise<void>`

Reply:
Do we do the same thing for `fetchRows`?

Reply:
yep !